app/helpers/votes_helper.rb

```diff
@@ -0,0 +1,96 @@
+module VotesHelper
+  def split_audit_hashes(input, model)
+    values = []
+    eval(input).each do |key, value|
```

Comment:
I think we should use a JSON-store in postgres instead.
http://blog.arkency.com/2014/10/how-to-persist-hashes-in-rails-applications-with-postgresql/
Scroll down to JSON.